.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Report a Bug
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Bug Description**
+A clear and concise description of what the bug is and how to reproduce it.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Browser (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Last upstream Slate commit (run `git log --author="Robert Lord" | head -n 1`):**
```

Comment:
We probably want a better solution for this... Maybe if we used a bot account to perform releases (merges from `dev` to `master` and tagging?) with a GitHub action of some kind?

Reply:
yeea it's less than ideal. if we switch to a node-based solution we could also include an npm script that would look up the closest match from a fresh pull of slatedocs/slate/master

Reply:
Most modern git clients should have regex capabilities enabled for author, so you could probably do:

`git log --author='\(Robert Lord\)\|\(lord\)\|\(Mike Ralphson\)\|\(Matthew Peveler\)' | head -n 1`